conf/application.conf

```diff
@@ -32,7 +32,9 @@ play.filters.csrf.contentType.whiteList = ["application/xml", "application/json"
 
 play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 play.http.errorHandler = "handlers.ErrorHandler"
-play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
+#play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
```

Comment:
Can you remove this comment?

And base on this change I think that we can get rid of `FrontendFilters` too